### DISCUSSION TITLE
Use legacy explorer

### DISCUSSION THREAD

**Discussion by ilyfairy** [valinet/ExplorerPatcher#3388] [General]
How can I use vivetool or modify the registry to return to the old version of explorer, or disable the rounded corners of the search bar?
After using ExplorerPatcher, explorer often crashes, but uninstalling ExplorerPatcher will restore the settings.
I want to do this without using ExplorerPatcher

**Comment by pyrates999** [comment]
try this: https://winaero.com/disable-rounded-corners-in-windows-11/

to fix the crashing, install the latest pre-release of EP: https://github.com/valinet/ExplorerPatcher/releases/tag/22621.3527.65.5_6f6666b
